### ISSUE TITLE
User shouldn't notice it when he gets deleted from contact list

### ISSUE THREAD

**Issue by ovalseven8** [Tox/Tox-Client-Standard#9]
Assume Alice and Bob are friends.

Bob deletes Alice from his contact list.
**Desired behaviour:** It looks for Alice like Bob is just going offline.

Should be **required** for all clients.

**Comment by JFreegman** [comment]
This is the default behaviour. In what circumstance would this not be the case?

**Comment by ovalseven8** [comment]
@JFreegman: Yes, it's the default behaviour. But the clients could implement that you can see it if you got deleted. Because of that, I thought it would make sense to add it to the Tox-Client-Standard.

**Comment by stal888** [comment]
The deleting client would have to be the one to send a notification about friend deletion.

**Comment by tux3** [comment]
If the protocol hasn't changed since irungentoo/toxcore#1427, in theory it's still possible for a client with patched toxcore to detect accurately when they get deleted by a friend, because empirically that's pretty much the only time we get a PACKET_ID_OFFLINE.

I'm not convinced we need to add that in the TCS, but if things haven't changed, it's technically possible.

**Comment by JFreegman** [comment]
> because empirically that's pretty much the only time we get a PACKET_ID_OFFLINE.

This seems like incorrect behaviour. I don't see why it should be necessary to send any information to a friend you're deleting. 

I'm going to make a pull request to remove this.

**Comment by nurupo** [comment]
@JFreegman https://github.com/irungentoo/toxcore/pull/1427#issuecomment-140600680

If I understand him right, if you are removing a friend from your friend list and you are also in the same group chat with the friend -- you won't timeout.

**Comment by JFreegman** [comment]
Oh so it's a hack due to the old groupchat implementation? Well let's get those new groupchats finished then.
